aeson-options.cabal

```diff
@@ -14,8 +15,6 @@ build-type:          Simple
 
 extra-source-files:  CHANGELOG.md
                    , README.md
-cabal-version:       >=2.0
-
 
```

Comment:
Well, if you are doing all this stuff, it would be good to also add `tested-with:` here, but I don't insist of course.